cnf-certification-test/manageability/suite.go

```diff
@@ -73,6 +73,9 @@ func containerPortNameFormatCheck(portName string) bool {
 }
 
 func testContainerPortNameFormat(env *provider.TestEnvironment) {
+	for protocolName := range env.ProtocolName {
+		allowedProtocolNames[env.ProtocolName[protocolName]] = true
```

Comment:
You can just reference your loop variable `protocolName` here instead of digging back through the map.